src/kirby/directives/theme-color/theme-color.directive.ts

```diff
@@ -5,9 +5,7 @@ import { ColorHelper } from '../../helpers/color-helper';
 
 @Directive({
   // tslint:disable-next-line:directive-selector
-  selector: `kirby-card[themeColor],
-             kirby-badge[themeColor],
-             kirby-icon[themeColor]`,
+  selector: `[themeColor]`,
```

Comment:
We did this for a reason (you couldn't know 😄 )

We did this because we didn't wanted the user of kirby to use `themeColor` on components that don't implement it yet and create a bug report on it. This way around the user would already on develop time see that a component don't support `themeColor`.

So let revert it. I know it is a hassle to maintain.

Reply:
Ok I see. But how about making the color style global, so we make it work for every scenario? @hafstad

Reply:
> Ok I see. But how about making the color style global, so we make it work for every scenario? @hafstad

I think we should keep that talk in an issue for it self. But for sure, lets have the talk.

Reply:
Let's do. I have fixed this now and will resolve this then.